BoxOffice/Model/BoxOfficeStructure.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  BoxOfficeStructure.swift
+//  BoxOffice
+//
+//  Created by Kim EenSung on 2/13/24.
+//
+
+import Foundation
+
+struct BoxOfficeResult: Decodable {
+    let boxOfficeResult: BoxOfficeDetail
+}
+
+struct BoxOfficeDetail: Decodable {
+    let boxofficeType: String
+    let showRange: String
+    let dailyBoxOfficeList: [MovieData]
+}
+
+struct MovieData: Decodable {
+    let index: String
+    let rank: String
+    let rankChangedAmount: String
+    let rankStatus: String
+    let movieCode: String
+    let movieName: String
+    let openDate: String
+    let salesAmount: String
+    let salesShare: String
+    let salesChangedAmount: String
+    let salesChangedPercentage: String
+    let salesAccmulated: String
+    let audienceCount: String
+    let audienceChangedAmount: String
+    let audienceChangedPercentage: String
+    let audienceAccmulated: String
+    let screenCount: String
+    let showCount: String
+    
+    enum CodingKeys: String, CodingKey {
+        case index = "rnum"
+        case rank
+        case rankChangedAmount = "rankInten"
+        case rankStatus = "rankOldAndNew"
+        case movieCode = "movieCd"
+        case movieName = "movieNm"
+        case openDate = "openDt"
+        case salesAmount = "salesAmt"
+        case salesShare
+        case salesChangedAmount = "salesInten"
+        case salesChangedPercentage = "salesChange"
+        case salesAccmulated = "salesAcc"
+        case audienceCount = "audiCnt"
+        case audienceChangedAmount = "audiInten"
+        case audienceChangedPercentage = "audiChange"
+        case audienceAccmulated = "audiAcc"
```

Comment:
```suggestion
        case salesAccumulated = "salesAcc"
        case audienceCount = "audiCnt"
        case audienceChangedAmount = "audiInten"
        case audienceChangedPercentage = "audiChange"
        case audienceAccumulated = "audiAcc"
```

간단한 스펠 체크는 아래 설정으로 수행해보세요. 캡처 하단의 Accmulated처럼 빨간 밑줄로 스펠이 잘못되었음을 파악할 수 있습니다.

<img width="1018" alt="image" src="https://github.com/tasty-code/ios-box-office/assets/69730931/ca9facd0-774b-4a14-b5fe-45e4866db620">

Reply:
꿀팁 감사드립니다!! 나름 오타 안낸다고 생각했었는데 부끄럽네요. 😅